salonA.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="static/css/main.css" />
+    <script type="module" src="src/JS/main.js"></script>
+    <title>Salong A</title>
+  </head>
+  <body>
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
+        <nav class="header-nav">
+          <button class="header-nav-toggle"></button>
+          <menu class="header-menu">
+            <li>
+              <a class="header-menu-item active" href="/index.html">Hem</a>
+            </li>
+            <li><a class="header-menu-item" href="#">Om oss</a></li>
+            <li><a class="header-menu-item" href="#">Biljettinfo</a></li>
+            <li><a class="header-menu-item" href="#">Presentkort</a></li>
+          </menu>
+        </nav>
+      </div>
+    </header>
+    <nav class="sec-cont">
+      <ul class="nav">
+        <li class="movies-nav">
+            <ul class="movies-nav-list">
+              <li class="movies-nav-element"><a href="#">Filmer <span class="arrow-top">&#9662;</span></a></li>
+              <li class="movies-nav-element"><a href="#">Komedi</a></li>
+              <li class="movies-nav-element"><a href="#">Action</a></li>
+              <li class="movies-nav-element"><a href="#">Drama</a></li>
+            </ul>
+        </li>
+        <li class="movies-nav">
+          <ul class="movies-nav-list">
+            <li class="movies-nav-element"><a href="#">Salonger <span class="arrow-top">&#9662;</span></a></li>
+            <li class="movies-nav-element"><a href="#">Salong A</a></li>
+            <li class="movies-nav-element"><a href="#">Salong B</a></li>
```

Comment:
href = should link to salons.html, salonA.html and salonB.html

Reply:
Fixed.